performance/font-loading.md

```diff
@@ -42,9 +54,9 @@ We believe that FOIT should be avoided as it is the worst experience for the use
 
 We highly recommend reading [Zach Leat's comprehensive guide](https://www.zachleat.com/web/comprehensive-webfonts/) to get a good understanding of the different techniques available. There is also a good [follow-up article](https://css-tricks.com/the-best-font-loading-strategies-and-how-to-execute-them/) that summarises Zach's findings.
 
-### Serve WOFF and WOFF2 font formats
+### WOFF and WOFF2 font formats
 
-WOFF2 has the superior compression. However, unfortunately [it does not cover all of our Enhanced browsers](https://caniuse.com/woff2) at the time of writing. Therefore consider serving both formats. 
+WOFF2 has a superior compression, and [is supported by all of our Enhanced browsers](https://caniuse.com/woff2), so there's no need to serve fonts in WOFF format.
```

Comment:
What's the impact for our other browsers?

Reply:
There shouldn't be any impact as:

* The only browser to ever being served WOFF fonts was Internet Explorer.
* Older versions of IE are not able to access our sites at all, as they lack TLS 1.2 support.
* Later versions of IE are classed as "Core" browsers, and we don't serve webfonts to them.